client/src/containers/RequestCard.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import LinearProgress from 'material-ui/LinearProgress';
+import RaisedButton from 'material-ui/RaisedButton';
+
+import { Link } from 'react-router-dom';
+import { differenceBetweenTwoDatesInDays } from '../utils/FormHelper';
+import './RequestCard.css';
+
+export default function RequestCard(props) {
+  const ordersLeft = props.numOrdered / props.numRequired * 100;
+  const countdown = differenceBetweenTwoDatesInDays(props.deadline);
+  return (
+    <div className="request-card">
+      <LinearProgress mode="determinate" value={ordersLeft} />
+      <div className="request-stat">
+        <span>
+          {props.numOrdered} <i className="material-icons cyan">favorite</i>
+        </span>
+        <span className="num-required">
+          {props.numRequired} <i className="material-icons cyan">restaurant</i>
+        </span>
+      </div>
+      <p>Countdown: {countdown} Days</p>
+      <div className="request-buttons">
+        <RaisedButton
+          onClick={() =>
+            props.orderRequestedMeal({ num: 1, requestId: props.requestId })}
```

Comment:
I hard coded the nun on purpose just to showcase how the arguments for the orderRequestMeal is going to look like